contributing.md

```diff
@@ -642,8 +642,8 @@ In general, we follow [Google's style guide](https://google.github.io/styleguide
 1. **Camel case**
 > In general, everything should use camel case. Exceptions include SDF element names, and protobuf variable names. Here is an [example](https://github.com/ignitionrobotics/ign-gazebo/blob/de5b025968c9cf0cfbfd8a852458482e87c70c6c/include/ignition/gazebo/SdfEntityCreator.hh#L64-L65).
 
-1. **Class function names**
-> Class functions must start with a capital letter, and capitalize every word.
+1. **Function names**
+> All functions (member functions and free functions) must start with a capital letter, and capitalize every word.
```

Comment:
Ah this also came up for me [last week](https://github.com/ignitionrobotics/ign-gazebo/pull/1394#discussion_r833523920), thanks for documenting.

That's not what we've been doing though. We've been using `camelBack` for free functions. [It's even in the clang-tidy configuration.](https://github.com/ignitionrobotics/ign-gazebo/blob/0996fe0ef676e1f3d237664e3dd6d15a6fb2464a/.clang-tidy#L5).

We can have a discussion about whether we'd like to change the policy, but if the goal here is to document the status quo, I believe most of the codebase has been following `camelBack` (I'm sure there are exceptions though).

Reply:
I've seen both `camelBack` and `CamelBack` for free function, but I just went through some of our code and I see that `CamelBack` is the exception, not the norm. I don't have an opinion on what it should be, I just thought we were following the GSG. I can change the document to explicitly state that free function should start with a lower case.

Reply:
Done in https://github.com/ignitionrobotics/docs/pull/239/commits/dff497a8366ce21e9779f962273e9761c6002dfd